### PR TITLE
update closure version

### PR DESCRIPTION
**Pull request by evmar**
Two commits, can review separately.

### CODE REVIEW

File: test/closure.ts [hunk 1]
```diff
@@ -64,7 +64,7 @@ function flagsToArgs(flags: Flags): string[] {
 
 /** Run the compiler, asynchronously returning a Result. */
 export function compile(options: Options, flags: Flags): Promise<Result> {
-  const jarPath = options.jarPath || require.resolve('google-closure-compiler/compiler.jar');
+  const jarPath = options.jarPath || require.resolve('google-closure-compiler-java/compiler.jar');
```
Comment (teppeis) [inline test/closure.ts]: The jar path is exported officialy https://github.com/google/closure-compiler-npm/blob/834b155375641b80f08464ffee38e58de6c4b764/packages/google-closure-compiler-java/index.js
```suggestion
  const jarPath = options.jarPath || require('google-closure-compiler-java');
```

Comment (evmar) [reply test/closure.ts]: Thanks!  Done.

**Comment by mprobst** [comment]
This looks reasonable, but did you see my PR https://github.com/angular/tsickle/pull/959 ?

**Comment by evmar** [comment]
Abandoning this, Martin already fixed this and I overlooked his change :~(

**Pull request closed without merging**